src/servers/ZoneServer2016/entities/npc.ts

```diff
@@ -17,16 +17,17 @@ import { BaseFullCharacter } from "./basefullcharacter";
 import { ZoneClient2016 } from "../classes/zoneclient";
 import { logClientActionToMongo } from "../../../utils/utils";
 import { DB_COLLECTIONS } from "../../../utils/enums";
-import { Items, MeleeTypes, StringIds } from "../models/enums";
+import { Items, StringIds } from "../models/enums";
 import { CommandInteractionString } from "types/zone2016packets";
 
 export class Npc extends BaseFullCharacter {
   health: number;
   npcRenderDistance = 80;
   spawnerId: number;
   deathTime: number = 0;
+  isZombie = false;
```

Comment:
Not a huge fan of this, zombies / wildlife should have their own class extending npc one or at least an Enum that replace the 2 booleans

Reply:
I edited this to use the NpcIds enums instead.